providers/aws/services/vpc/vpc_flow_logs_enabled/vpc_flow_logs_enabled_test.py

```diff
@@ -0,0 +1,127 @@
+from unittest import mock
+
+from boto3 import client
+from moto import mock_ec2
+
+AWS_REGION = "us-east-1"
+ACCOUNT_ID = "123456789012"
+
+
+class Test_vpc_flow_logs_enabled:
+    @mock_ec2
+    def test_vpc_only_default_vpcs(self):
+        from providers.aws.lib.audit_info.audit_info import current_audit_info
+        from providers.aws.services.vpc.vpc_service import VPC
+
+        current_audit_info.audited_partition = "aws"
+
+        with mock.patch(
+            "providers.aws.services.vpc.vpc_flow_logs_enabled.vpc_flow_logs_enabled.vpc_client",
+            new=VPC(current_audit_info),
+        ):
+            # Test Check
+            from providers.aws.services.vpc.vpc_flow_logs_enabled.vpc_flow_logs_enabled import (
+                vpc_flow_logs_enabled,
+            )
+
+            check = vpc_flow_logs_enabled()
+            result = check.execute()
+
+            assert (
```

Comment:
Yo have to check status, status_extended and resource id for each one.

Reply:
We cannot do it since they are the deafult ones.